report_xlsx/controllers/main.py

```diff
@@ -52,6 +52,7 @@ def report_routes(self, reportname, docids=None, converter=None, **data):
     def report_download(self, data, context=None, token=None):
         requestcontent = json.loads(data)
         url, report_type = requestcontent[0], requestcontent[1]
+        reportname = "???"
```

Comment:
Actually the issue seems to be referencing `reportname` in the except block if an exception occurred and the report wasn't XLSX.

I would instead move the try except inside the `if report_type == "xlsx"` check, and avoid catching any exception for other report types here.